views/deductions/donations-amount.pug

```diff
@@ -0,0 +1,31 @@
+extends ../base
+
+block variables
+  -var title = __('Enter your Charitable Donations')
+
+block content
+
+  h1 #{title}
+
+  div
+    p #{__('Please enter your charitable donations into the fields below.')}
+
+  form.cra-form(method='post')
+    div(class={['has-error']: errors && errors.donationsAmount})
+      label#donations__label(for='donationsAmount') #{__('Total donations made to registered charities')}
+      if errors
+        +validationMessage(errors.donationsAmount.msg, 'donations')
+      span#donations__unit.input-with-unit__unit(for='donationsAmount') $
+      input.w-1-2.input-with-unit#donations(name='donationsAmount', type='text', placeholder="0.00", autocomplete='off' aria-describedby=(errors && errors.donations ? 'donations-error' : false) aria-labelledby="donations__label donations__unit")
+    div
+      details.
+        <summary><span>#{__('Help with charitable donations')}</span></summary>
+        <p>#{__('Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.')}</p>
+
+    input#redirect(name='redirect', type='hidden', value='/personal/maritalStatus')
```

Comment:
Ah, just noticed. According to our designs, it goes:

- Financial info 
- RRSP 
- Marital status
- Spouse and dependents (which we don't have)
- Political contributions
- Charitable donations

Can we put this after the marital status check?

Reply:
for sure